src/DynamoCore/DynamoCore.csproj

```diff
@@ -379,6 +356,32 @@ limitations under the License.
   <ItemGroup>
     <Resource Include="BuiltInAndOperators\BuiltIn.Migrations.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="DynamoVisualProgramming.LibG_225_0_0" GeneratePathProperty="true">
+      <Version>*</Version>
+      <IncludeAssets>all</IncludeAssets>
+      <ExcludeAssets>compile</ExcludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="DynamoVisualProgramming.LibG_226_0_0" GeneratePathProperty="true">
+      <Version>*</Version>
+      <IncludeAssets>all</IncludeAssets>
+      <ExcludeAssets>compile</ExcludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+    <PackageReference Include="DynamoVisualProgramming.LibG_227_0_0" GeneratePathProperty="true">
+      <Version>*</Version>
+    </PackageReference>
+    <PackageReference Include="Greg">
+      <Version>*</Version>
```

Comment:
is this really what we want? I understand with LibG it's a pain to update all the references, but Greg is only referenced from a few projects, and theres only one binary... it feels kind of dangerous to have our builds potentially pull different versions of dependencies when they build on build machines than they did when we build them locally...

@alfredo-pozo will the logs contain what versions are used?

Reply:
@mjkkirschner   On the logs you will see all the versions installed (Greg and LibG), feel free to take a look on the PR job console output https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/pullRequestValidation/1635/consoleText , not sure if I understand your concern, because the versions that we are going to use are coming from one source only, and this version will change only if changes that were approved and reviewed on LibG and Greg will be merged and deployed from the main branches, and when that happens, on our current process we need to update all the references to use the new versions and they supposed to be stable version (new PR each time), for me that will be an issue if the nuget packages were updated without any code review process and the PR validation jobs that we currently have.

Reply:
![image](https://user-images.githubusercontent.com/47230272/96507107-56d43a00-121e-11eb-8f94-1444483e3e57.png)